ui/design/src/components/Editor/index.tsx

```diff
@@ -705,7 +710,12 @@ const EditorBox: React.FC<IEditorBox> = React.forwardRef((props, ref) => {
           )}
           {withMeter && <EditorMeter counter={letterCount} maxValue={MAX_LENGTH} />}
           {showCancelButton && (
-            <Button secondary={true} label={cancelButtonLabel} onClick={onCancelClick} />
+            <StyledCancelButton
+              secondary={true}
+              slimBorder={true}
+              label={cancelButtonLabel}
+              onClick={onCancelClick}
+            />
```

Comment:
Within the `Button` component, there is a `StyledButton` and if you include say a flag called `accentBorder` inside the `Button` component then you can use it to toggle an accent border color on and off on `StyledButton`. That way the button can be reusable elsewhere whenever you want accent border. Otherwise you are going to override it's style everywhere you need this accent border color.

Reply:
Thanks @didd for pointing it out. I'm looking into it :)